github/Rate.py

```diff
@@ -63,10 +63,18 @@ def reset(self):
         """
         return self._reset.value
 
+    @property
+    def used(self):
```

Comment:
```suggestion
    def used(self) -> int:
```

Reply:
I believe that typing is handled by `Rate.pyi`

Reply:
we are going to merge `Rate.pyi` back to `Rate.py`, so it would be helpful to also add inline type.